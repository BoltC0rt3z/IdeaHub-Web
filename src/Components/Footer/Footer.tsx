import React from 'react';
import { Layout, Row } from 'antd';

const { Footer: AntFooter } = Layout;

const Footer: React.FC = () => {
    return <AntFooter><Row type="flex" justify="space-around">&copy; IdeaHub</Row></AntFooter>;
};

export default Footer;
