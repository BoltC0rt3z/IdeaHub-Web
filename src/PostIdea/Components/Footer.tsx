import React from 'react';
import { Row, Col } from 'antd';

import { Button } from '../../Components';

import styles from './Footer.module.scss';

interface Props {
  onCancel: () => void;
  onOk: () => void;
}

const Footer: React.FC<Props> = (props) => {
  const { onCancel, onOk } = props;

  return (
    <Row type="flex" className={styles.footer}>
      <Col>
        <Button size="large" type="primary" className={styles.footer} onClick={onOk}>
          Post Idea
        </Button>
      </Col>
      <Col className={styles.cancel} onClick={onCancel}>
        Cancel
      </Col>
    </Row>
  );
};

const MobileFooter: React.FC<Props> = (props) => {
  const { onCancel, onOk } = props;

  return (
    <>
      <Col className={styles.mobileFooter}>
        <Button type="primary" className={styles.button} onClick={onOk} size="large">
          Post Idea
        </Button>
      </Col>
      <Col>
        <Button type="danger" className={styles.button} onClick={onCancel} size="large">
          Cancel
        </Button>
      </Col>
    </>
  );
};

export { MobileFooter };

export default Footer;
