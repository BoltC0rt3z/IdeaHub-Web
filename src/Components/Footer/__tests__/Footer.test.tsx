import React from 'react';
import { ReactWrapper, mount } from 'enzyme';

import Footer from '../Footer';

describe('Tests for the footer', () => {
    const wrapper: ReactWrapper = mount(<Footer />);

    it('should test for footer', () => {
        expect(wrapper).toMatchSnapshot();
    });
});