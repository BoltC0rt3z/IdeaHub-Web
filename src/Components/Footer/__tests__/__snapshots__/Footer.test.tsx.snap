// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests for the footer should test for footer 1`] = `ReactWrapper {}`;
